data-serving/samples/line-list.json

```diff
@@ -168,6 +168,7 @@
         "id": "def456",
         "url": "https://www.colorado.gov/pacific/cdphe/news/10-new-presumptive-positive-cases-colorado-cdphe-confirms-limited-community-spread-covid-19"
     },
+    "outcome": "Pending",
```

Comment:
outcome was intentionally removed and folded into events: https://docs.google.com/document/d/1fMPiwohsN2vDyeW7TafPLvAUij4bi49KjB4pTV7kJco/edit#heading=h.5qv1o3y0h3l1

(it was removed from the mongodb schema too: https://github.com/open-covid-data/healthmap-gdo-temp/pull/63/files)

you could probably find the event they had as the outcome with something like this:

`this.events.find((e: Event) => e.date == null)`

Reply:
sorry, that code snippet will only work for real data (where 'outcome' has no date) -- in the sample data, I did add a date to the outcome

Reply:
Reverted this file change